ulohy_07_hangman/tests.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import validation
+import ai
+import hangman
+
+
+# Testování vstupu
+def test_letter_in_word():
+    assert validation.check_string("o", "kočka") == True
```

Comment:
`bool` je lepší porovnávat přes klíčové slovíčko `is` podobně jako v případě `None`. Zde to je nicméně redundantní. Úplně by stačilo, mít tam pouze `assert validation.check_string("o", "kočka")` protože, sám `assert` provádí onu kontrolu vůči `True`. I `pycodestyle` na to upozorňuje :)